admin-system/backend/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  admin-server:
+    image: ghcr.io/computing-collective/3fa-backend:latest
+    volumes:
+      - type: bind
+        source: "instance"
```

Comment:
On my deployment side I prefer to keep it consistent: 
`/nfs/3fa/data` if possible, unless "instance" is a requirement

Reply:
`instance` is not a requirement but for consistency and for anyone else running this it makes more sense since `/nfs/3fa/data` won't exist and will return an error due to nfs being a special file system. Using `instance` as the default also has the benefit that if it is run from the `admin-system/backend` folder then runs between Docker and local pipenv will share the same data.

As for deployment you can simply change the value after and continue to git pull to grab latest changes without any conflicts.

Reply:
seems good